.gitmodules

```diff
@@ -42,7 +42,7 @@
 	url = https://github.com/pulp-platform/axi_riscv_atomics.git
 [submodule "corev_apu/riscv-dbg"]
 	path = corev_apu/riscv-dbg
-	url = https://github.com/pulp-platform/riscv-dbg.git
+	url = https://github.com/JeanRochCoulon/riscv-dbg.git
```

Comment:
I would prefer to not fork if possible.
IMO, it seems better to modify riscv-dbg according to https://github.com/pulp-platform/riscv-dbg/pull/83

Reply:
I think @JeanRochCoulon and I agree that the problem he was seeing because of a bug in the debug unit that should have been fixed by later commits. Can you try updating to the latest `riscv-dbg` module? We need to merge this before we can merge this PR as the public CVA6 can't depend on "private" forks of members.

Reply:
With latest version of `riscv-dbg`, the problem disappears. PR has been adapted accordingly.